### PR TITLE
Add support for 1.19.3

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

**Comment by NotMyFault** [comment]
Some preliminary testing revealed no issues. Regeneration, history, schematics are working fine so far.

If there's nothing more to address and nobody reports major issues, I'd propose we go ahead and merge it by next week.

**Review by x9nico — APPROVED** [review]
Looks good

File: worldedit-bukkit/adapters/adapter-1_19_3/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+repositories {
+    maven {
+        name = "PaperMC"
+        url = uri("https://repo.papermc.io/repository/maven-public/")
+    }
+}
+
+dependencies {
+    // https://papermc.io/repo/service/rest/repository/browse/maven-public/io/papermc/paper/dev-bundle/
+    paperDevBundle("1.19.3-R0.1-20221210.010342-6")
```
Comment (NotMyFault) [inline worldedit-bukkit/adapters/adapter-1_19_3/build.gradle.kts:16]:
```suggestion
    paperDevBundle("1.19.3-R0.1-20221211.165921-14")
```

**Review by PierreSchwang — APPROVED** [review]
I think the adapters need quite a bit of rework, but that's more of a general issue with the codebase as it is now lol
Take my review with a grain of salt, as I've just tested regeneration and basic set-block while making my changes.

**Pull request merged**